hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/contracts/operations/HederaEvmCreate2Operation.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.evm.contracts.operations;
+
+import javax.inject.Inject;
+import org.hyperledger.besu.datatypes.Address;
+import org.hyperledger.besu.evm.frame.MessageFrame;
+import org.hyperledger.besu.evm.gascalculator.GasCalculator;
+
+public class HederaEvmCreate2Operation extends AbstractEvmRecordingCreateOperation {
+    @Inject
+    public HederaEvmCreate2Operation(final GasCalculator gasCalculator) {
+        super(0xF5, "ħCREATE2", 4, 1, 1, gasCalculator);
+    }
+
+    @Override
+    protected boolean isEnabled() {
+        return false;
+    }
+
+    @Override
+    protected long cost(final MessageFrame frame) {
+        return gasCalculator().create2OperationGasCost(frame);
+    }
+
+    @Override
+    protected Address targetContractAddress(MessageFrame frame) {
+        return null;
```

Comment:
We should have the implementation extracted from `HederaCreate2Operation` to here.